Sprint-1/exercises/initials.js

```diff
@@ -4,3 +4,5 @@ let lastName = "Johnson";
 
 // Declare a variable called initials that stores the first character of each string.
 // This should produce the string "CKJ", but you must not write the characters C, K, or J in the code of your solution.
+var initials =
+  firstName.slice(0, 1) + middleName.slice(0, 1) + lastName.slice(0, 1);
```

Comment:
There is a simpler syntax to retrieve the first character from a string.

Reply:
Strings in JavaScript can be treated like arrays of characters, so i have updated using   firstName[0] + middleName[0] + lastName[0], now i think more accrued